### PR TITLE
 qa/vstart_runner: log stdout and stderr of every commmand by default 

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
since teuthology does it too.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by joscollin — APPROVED** [review]

**Comment by rishabh-d-dave** [comment]
Marking DNM since this PR can take some improvement.

**Comment by rishabh-d-dave** [comment]
@joscollin I've updated the PR branch to include return value of mount command in logs and remove few extra/redundant variables.

**Comment by joscollin** [comment]
> @joscollin I've updated the PR branch to include return value of mount command in logs and remove few extra/redundant variables.

I don't see any issues with the new change. It looks better now.

**Comment by rishabh-d-dave** [comment]
@joscollin Cool, thanks for the quick review!

**Review by lxbsz — APPROVED** [review]

**Review by kotreshhr — APPROVED** [review]
Looks good. Please rebase.

File: qa/tasks/cephfs/kernel_mount.py [hunk 1]
```diff
@@ -67,19 +67,21 @@ def mount(self, mntopts=[], check_status=True, **kwargs):
...
+                     f'{mount_proc.returncode}')
             if check_status:
+                log.debug(f'stdout -\n{mount_proc.stdout.getvalue()}')
+                log.debug(f'stderr -\n{mount_proc.stderr.getvalue()}')
                 raise
             else:
-                return (e, mountcmd_stdout.getvalue(),
-                        mountcmd_stderr.getvalue())
+                return (e, mount_proc.stdout.getvalue(),
+                        mount_proc.stderr.getvalue())
```
Comment (rishabh-d-dave) [inline qa/tasks/cephfs/kernel_mount.py:84]: On second look, this part is buggy because `mount_proc` won't be initialized when exception occurs.

**Review by rishabh-d-dave — COMMENTED** [review]
Also, I am trying to make this patch more generic. Not sure, if that can be done, I'll try it out and update the PR.

**Pull request closed without merging**